.gitignore

```diff
@@ -5,3 +5,4 @@ ui-debug.log
 .idea
 node_modules/
 hosting/build
+hosting/dist
```

Comment:
Is this used in parallel to `hosting/build`, or does it replace it? Ideally I'd like to continue using `build` instead of `dist` for consistency with other projects if we can.

Reply:
Updated the vite config to output to build instead of dist and removed the additional ignore entry.